src/common/sort/sort_state.cpp

```diff
@@ -268,7 +271,7 @@ unique_ptr<RowDataBlock> LocalSortState::ConcatenateBlocks(RowDataCollection &ro
 	// Create block with the correct capacity
 	auto buffer_manager = &row_data.buffer_manager;
```

Comment:
nit: can this be turned into a reference

Reply:
```cpp
LocalSortState::LocalSortState() : initialized(false) {
	if (!Radix::IsLittleEndian()) {
		throw NotImplementedException("Sorting is not supported on big endian architectures");
	}
}
```
Complains. 🤔